MessagingServiceTests/MessagingServicePOSTTests.cs

```diff
@@ -0,0 +1,975 @@
+﻿using Microsoft.Owin.Testing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json.Linq;
+using NuGet.Services.Messaging;
+using NuGet.Services.Metadata.Catalog.Persistence;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MessagingServiceTests
+{
+    /// <summary>
+    /// ContactOwners tests
+    /// </summary>
+    [TestClass]
+    public class ContactOwnersTests
+    {
+        private static TestServer _server;
+        private static TestServer _server_noStorage;
+        // private static TestServer _server_noAAD;
+
+        private static StorageManager _storageManager;
+        private static StorageManager _storageManagerFake;
+
+        private const string TestJSONPath = @"C:\Users\rebro\Documents\GitHub\NuGet\NuGet.Services.Email\MessagingServiceTests\sampleJSON\ContactOwners.json";
```

Comment:
Thanks for pointing that out, I'll fix that now.  I will switch to using relative paths so the tests are portable.